### PR TITLE
Make Django views names more informative

### PR DESCRIPTION
**Pull request by mofr**
It's useful for New Relic APM

### CODE REVIEW

File: winter/django.py [hunk 1]
```diff
@@ -53,6 +54,10 @@ class WinterView(rest_framework.views.APIView):
         permission_classes = (IsAuthenticated,) if is_authentication_needed(component) else ()
         throttle_classes = create_throttle_classes(component, routes)
 
+    # It's useful for New Relic APM
+    WinterView.__module__ = controller_class.__module__
```
Comment (berenda-andrey) [inline winter/django.py]: it is better to use update_wrapper

Comment (berenda-andrey) [reply winter/django.py]: from functools
this method moves doc also

Comment (mofr) [reply winter/django.py]: image

Comment (berenda-andrey) [reply winter/django.py]: seems need to update version

Comment (pristupa) [reply winter/django.py]: version of what?

Comment (berenda-andrey) [reply winter/django.py]: не правильно комент написал, то есть не туда, самого winter на 2.0.1

Comment (berenda-andrey) [reply winter/django.py]: а то не получится зарелизить

**Comment by berenda-andrey** [comment]
seems need to update winter version

**Comment by mofr** [comment]
Closes #131

**Pull request merged**